### DISCUSSION TITLE
How can I change the PAUSE macro position?

### DISCUSSION THREAD

**Discussion by scmanjarrez** [jschuh/klipper-macros#126] [Q&A] [answered]
Hi, thanks for your work. I really like your macros. 

I'd like to ask how can I change the park position to be at x0 y235 (y limit) instead of x0 y0.

**Comment by jschuh** [comment, marked as answer]
All the config parameters are described in globals.cfg. You just copy the relevant variable from there into the `[gcode_macro _km_options]` section in your own config and set the value to whatever you like. The base parking positions are here: https://github.com/jschuh/klipper-macros/blob/b53e538aef7ec48c73291f566016ed523e60d26b/globals.cfg#L40-L45

There's also one special parking position for the Y axis, used at the end of the print (where the toolhead is most often parked in the rear): https://github.com/jschuh/klipper-macros/blob/b53e538aef7ec48c73291f566016ed523e60d26b/globals.cfg#L64-L65

There's a whole mess of config options specific to `PRINT_START`/`PRINT_END`, so they're [documented in much more detail in the readme](https://github.com/jschuh/klipper-macros#print-start-and-end).

**Reply by scmanjarrez** [reply]
Thank you!
